packages/sitecore-jss-react/src/components/VisitorIdentification.tsx

```diff
@@ -3,27 +3,37 @@ import { withSitecoreContext } from '../enhancers/withSitecoreContext';
 
 interface VIProps {
   sitecoreContext: {
+    pageEditing?: boolean;
     visitorIdentificationTimestamp?: string;
   };
 }
 
 let emittedVI = false;
-let VIComponent: React.SFC<VIProps> = ({ sitecoreContext }) => {
+const VIComponent: React.FC<VIProps> = ({ sitecoreContext }) => {
   if (
-    !emittedVI &&
-    typeof document !== 'undefined' &&
-    sitecoreContext.visitorIdentificationTimestamp
+    emittedVI ||
+    typeof document === 'undefined' ||
+    sitecoreContext.pageEditing ||
```

Comment:
@ambrauer So, in Experience Editor this warning also exists?

Reply:
Nvm, I understood that it shouldn't be invoked in EE

Reply:
Need remember to add in release notes that pageEditing will be checked inside component

Reply:
@sc-illiakovalenko Actually removed sitecoreContext.pageEditing check after further testing. This is not necessary as sitecoreContext.visitorIdentificationTimestamp is always undefined in editing/preview mode. I believe when I noticed this issue it was during editing, but the main issue was adding the meta tag w/ VI timestamp at render time (which has been resolved).